linux/net/rina/dt-utils.c

```diff
@@ -953,7 +953,7 @@ int common_efcp_pdu_send(struct efcp * efcp,
 
 	/* Remote flow case */
 	if (pci_source(pci) != pci_destination(pci)) {
-	        if (rmt_send(rmt, pci, pdu)) {
```

Comment:
The signature of ```rmt_send``` included the PCI just to avoid having to retrieve it again, but ok, I was about to change this once.